src/taipy/core/config/__init__.py

```diff
@@ -68,12 +68,12 @@
     ],
 )
 _inject_section(
-    PipelineConfig,
-    "pipelines",
-    PipelineConfig.default_config(),
+    SequenceConfig,
+    "sequences",
+    SequenceConfig.default_config(),
     [
-        ("configure_pipeline", PipelineConfig._configure),
-        ("set_default_pipeline_configuration", PipelineConfig._set_default_configuration),
+        ("configure_sequence", SequenceConfig._configure),
+        ("set_default_sequence_configuration", SequenceConfig._set_default_configuration),
     ],
 )
```

Comment:
This part should not exist anymore, no?

Reply:
we're postponing the removal of Sequence config, repo and model to make sure everything is still working fine without any dependency on those classes. But you're right, contest is also part of the dependency. Will remove it now. The plan is to remove them in PR https://github.com/Avaiga/taipy-core/pull/739